### PR TITLE
feat(Table): support virtual components.body

### PR DESCRIPTION
**Pull request by linxianxi**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

https://github.com/react-component/table/pull/1069

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Table support components.body when virtual      |
| 🇨🇳 Chinese |    Table 支持 virtual 下的 components.body       |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -153,7 +153,7 @@
     "rc-slider": "~10.5.0",
     "rc-steps": "~6.0.1",
     "rc-switch": "~4.1.0",
-    "rc-table": "~7.37.0",
+    "rc-table": "~7.38.0",
```
Comment (yoyo837) [inline package.json:156]: 等等把 master 同步一次，然后把上午豆酱合并的PR 的文档调整一下。

Comment (yoyo837) [reply package.json:156]: rebase 一下

Comment (linxianxi) [reply package.json:156]: 貌似 dumi lint 挂了

File: components/table/index.zh-CN.md [hunk 2]
```diff
@@ -385,3 +385,7 @@ return <Table rowKey={(record) => record.uid} />;
 ### 如何自定义渲染可选列的勾选框（比如增加 Tooltip）？
 
 自 `4.1.0` 起，可以通过 [rowSelection](https://ant.design/components/table-cn/#rowselection) 的 `renderCell` 属性控制，可以参考此处 [Demo](https://codesandbox.io/s/table-row-tooltip-v79j2v) 实现展示 Tooltip 需求或其他自定义的需求。
+
+### 为什么 components.body.wrapper 在 virtual 开启时会崩溃？
```
Comment (zombieJ) [inline components/table/index.zh-CN.md:389]: > ### 为什么 `components.body` 在 `virtual` 开启时会报错？
> 因为虚拟表格需要通过 `ref` 计算单元格位置信息，你需要。。。

英文同样改一下。不要透露出 `rc-virtual-list` ，这是底层的库，用户感知不到的。

Comment (linxianxi) [reply components/table/index.zh-CN.md:389]: done

Comment (zombieJ) [reply components/table/index.zh-CN.md:389]: 标题没改。。。

Comment (linxianxi) [reply components/table/index.zh-CN.md]: 改好了。我感觉应该要写 components.body.wrapper 吧，components.body 不太准确

**Review by zombieJ — APPROVED** [review]

**Pull request merged**